the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
 			<scope>test</scope>
 		</dependency>
-		
```

Comment:
public class TheInternet:
WebDrivers should be located in the Resources folder instead so we can get the path dynamically. Currently the path is hardcoded.